src/spacegame3/SpaceGame.java

```diff
@@ -80,29 +94,45 @@ public void giveSceneTo(SizableScene scene){
         stage.setScene(scene);
     }
 
+    /*
+        *main method - runs + launches game
+    */
     public static void main(String[] args) {
         launch(args);
     }
 
+    //getter for game started
     public boolean gameStarted() {
         return gameStarted;
     }
 
-
+    /*
+        *sets the planet screen to have landed on the parameter (a CelestialBody object)
+        *returns planet screen
+    */
     public SizableScene getPlanetScreen(CelestialBody planet) {
         planetScreen.setLandedOn(planet);
         return planetScreen;
     }
 
+    /*
+        *saves the current scene
+        *transfers to the start screen
+    */
     public void showStartScreen(SizableScene scene) {
         previousScene = scene;
         giveSceneTo(startScreen);
     }
 
+    //gets previous scene
     public SizableScene previousScene(){
         return previousScene;
     }
 
+    /*
+        *will send back to current screen if that is not null
+        * if it is null,
+    */
```

Comment:
use /** */ for methods (just repeating for each one)

also this is not what this method does ;) i am giving you another try before i give the answer :)